client/src/pages/CreateLesson/CreateLesson.js

```diff
@@ -57,15 +73,36 @@ const CreateLesson = ({ navigation }) => {
 
   const body = (
     <>
-      <Text>Give your lesson a name</Text>
+            <View style={styles.container}>
+          <View style={styles.textRow}>
+            <Foundation name="lightbulb" size={20} color={colors.blue.dark} />
+            <Text
+              fontSize="md"
+              paddingBottom={2}
+              fontFamily="heading"
+              fontWeight="regular"
+              fontStyle="normal"
+              color={colors.blue.dark}
+            >
+              {' '}
+              Suggestion{' '}
+            </Text>
+          </View>
+          <Text color={colors.blue.dark} fontSize="md">
+            When creating a unit, think about how it will be used. More text
```

Comment:
Can you update this text so it says, "When creating a lesson, think about which vocab items will be associated with the lesson and how it will be used?" Right now, the text looks to be default text from the Figma designs.